### PR TITLE
Fix blurry font on retina display for the demo application

### PR DESCRIPTION
**Pull request by ArnaudBienner**
I had to fix this same problem many times, on all Qt applications I worked on on Mac (and I'm not the only one e.g. https://trac.macports.org/ticket/36410)
Even if it's a demo application, this hurt my little eyes everytime I tested something. :dizzy_face:

### CODE REVIEW

File: Info.plist.in [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrincipalClass</key>
+	<string>NSApplication</string>
```
Comment (ArnaudBienner) [inline Info.plist.in]: The important thing is those two lines.
Except this, the file is similar to MacOSXBundleInfo.plist.in Info.plist.in which is the default one used when no plist file is specified.
I don't remember the exact reason why this is needed, but it is something like a way for the application to tell it is "retina ready"

**Comment by ArnaudBienner** [comment]
Since this is a very simple change, I will abuse my new power and merge this pull request myself.

File: Info.plist.in (hunk 1 above)
Comment (mikemcquaid) [inline Info.plist.in]: :+1:

Comment (mikemcquaid) [inline Info.plist.in]: Maybe add this as an XML comment in a future PR?

Comment (ArnaudBienner) [inline Info.plist.in]: OK, will do :-)

**Pull request merged**